nautobot/extras/tables.py

```diff
@@ -493,3 +494,33 @@ class Meta(BaseTable.Meta):
             "content_type",
             "description",
         )
+
+
+class CustomFieldTable(BaseTable):
+    pk = ToggleColumn()
+    slug = tables.Column(linkify=True, accessor="name")
```

Comment:
Possibly add a TODO comment here referencing https://github.com/nautobot/nautobot/issues/464 and the plan to eventually replace `name` with `slug` on the model?